lib/navigation/navigator.hpp

```diff
@@ -32,9 +32,9 @@ class Navigator : public INavigator {
   /**
    * @brief preprocesses imu data and updates trajectory
```

Comment:
one more imu -> accelerometer then good to go

Reply:
damn, forgot to check the other files